src/apps/material-search/MaterialSearchListResults.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { FC } from "react";
+import { WorkId } from "../../core/utils/types/ids";
+import { Cover } from "../../components/cover/cover";
+import MaterialSearchLoading from "./MaterialSearchLoading";
+import { flattenCreators } from "../../core/utils/helpers/general";
+import { useText } from "../../core/utils/text";
+import useInfiniteScrollLoading from "./useInfiteScrollLoading";
+import { SearchWithPaginationQuery } from "../../core/dbc-gateway/generated/graphql";
+
+type MaterialSearchListResultsProps = {
+  data: SearchWithPaginationQuery["search"]["works"];
+  isLoading: boolean;
+  onWorkIdSelect: (workId: WorkId) => void;
+  loadMore: () => void;
+  hitCount: number;
+};
+
+const MaterialSearchListResults: FC<MaterialSearchListResultsProps> = ({
+  data,
+  isLoading,
+  onWorkIdSelect,
+  loadMore,
+  hitCount
+}) => {
+  const t = useText();
+
+  const { containerRef, lastItemRef, handleKeyDown, handleFocus } =
+    useInfiniteScrollLoading({
+      data,
+      isLoading,
+      loadMore,
+      hitCount,
+      onWorkIdSelect
+    });
+
+  if (isLoading && data.length === 0) {
+    return (
+      <div className="material-search__list">
+        <div className="material-search__list-header">
+          <MaterialSearchLoading />
+        </div>
+      </div>
+    );
+  }
+
+  if (!data || data.length === 0) return null;
+
+  return (
+    <div className="material-search__list" ref={containerRef}>
+      <div className="material-search__list-header">
+        {t("materialSearchAmountOfResultsText")}:
+        <span className="material-search__list-header--result-count">
+          {hitCount}
+        </span>
+      </div>
+      <ol className="material-search__list-results">
+        {data.map((work, index) => {
+          const authors = flattenCreators(work.creators);
+          const isLastItem = index === data.length - 1;
+
+          return (
+            <li
+              className="material-search__list-result"
+              key={work.workId}
+              ref={isLastItem ? lastItemRef : null}
+            >
+              <button
+                className="material-search__list-result-button"
+                type="button"
+                tabIndex={0}
+                onClick={(e) => {
+                  e.preventDefault();
```

Comment:
Is `e.preventDefault()` needed when the type is not `type="submit"`?

Reply:
It will submit the drupal form if it is not there i think.